tests/writer_util.py

```diff
@@ -152,8 +152,8 @@ def _convert_clob_pairs(clob_pairs):
         (_D('0e-15'), b'0d-15'),
         (_D('-1e1000'), b'-1d+1000'),
         (_D('-4.412111311414141e1000'), b'-4.412111311414141d+1000'),
-        (_D('1.1999999999999999555910790149937383830547332763671875e0'),
-            b'1.1999999999999999555910790149937383830547332763671875'),
+        # (_D('1.1999999999999999555910790149937383830547332763671875e0'),
+        #     b'1.1999999999999999555910790149937383830547332763671875'),
```

Comment:
As discussed offline, C extension only supports 34 decimal digits. I comment this for beta version for now.